packages/builder/src/types/app.ts

```diff
@@ -10,3 +12,21 @@ export type AppTemplate =
       background: string
       icon: string
     }
+
+export interface AppIdentifierMetadata {
```

Comment:
Can we keep types in `@budibase/types`? I think having a single type import of `import type {...} from @budibase/types` is a lot easier than having to import types from multiple places.